lms/static/js/student_account/views/RegisterView.js

```diff
@@ -1,100 +1,98 @@
-var edx = edx || {};
-
-(function($, _, gettext) {
+;(function (define) {
     'use strict';
+    define(['jquery', 'underscore', 'js/student_account/views/FormView'], function($, _, FormView) {
+
+        var RegisterView = FormView.extend({
```

Comment:
Instead of assigning the view to a variable and returning that you can just return the view, i.e.
```
return FormView.extend({
```